From 8585da65b0e0504bf51297ac04d306754cfd3181 Mon Sep 17 00:00:00 2001
From: li zhuangzhi <zhuangzhi.li@intel.com>
Date: Wed, 9 Oct 2019 14:17:27 +0800
Subject: [PATCH] Extend the timeout value for uevent listener

This is WA for fixing boot issue on 3th ~ 5th old platforms.

Tracked-On:
Signed-off-by: li zhuangzhi <zhuangzhi.li@intel.com>
---
 init/first_stage_mount.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/first_stage_mount.cpp b/init/first_stage_mount.cpp
index 3e76556..f332100 100644
--- a/init/first_stage_mount.cpp
+++ b/init/first_stage_mount.cpp
@@ -296,7 +296,7 @@ bool FirstStageMount::InitRequiredDevices() {
                   << ": partition(s) not found in /sys, waiting for their uevent(s): "
                   << android::base::Join(required_devices_partition_names_, ", ");
         Timer t;
-        uevent_listener_.Poll(uevent_callback, 10s);
+        uevent_listener_.Poll(uevent_callback, 30s);
         LOG(INFO) << "Wait for partitions returned after " << t;
     }
 
-- 
1.9.1

